app/gen-server/lib/homedb/ServiceAccountsManager.ts

```diff
@@ -75,7 +75,7 @@ export class ServiceAccountsManager {
    *  - the information from the users table
    *  - the login information
    *
-   * @param serviceAccountLogin The service account email
+   * @param serviceAccountId The service account email
```

Comment:
The comment still talks about an email even though `serviceAccountId` is a number.
```suggestion
   * @param serviceAccountId The service account ID
```

The same issue exists for other methods below.